development.ini

```diff
@@ -47,7 +47,7 @@ atramhasis.session_factory.secret = itsaseekreet
 atramhasis.dump_location = %(here)s/datadumps
 
 # Location of rdf2hdt executable
-atramhasis.rdf2hdt = /link/to/rdf2hdt
+atramhasis.rdf2hdt = /usr/local/bin/rdf2hdt
```

Comment:
I doubt if we want to change this. This repo can be downloaded by users using all kind of machines. `/usr/local/bin/rdf2hdt` makes the suggestion that the file will always be on that location. On f.e. Windows machines that won't be the case. `/link/to` is more clear that this is a setting that the user will need to change.

Or we could use `atramhasis.rdf2hdt = /usr/local/bin/rdf2hdt` and adjust the comment above, something like `# Location of rdf2hdt executable. Change this location to the path where the rdf2hdt can be found on your machine`